tests/unit/test_task8_9.py

```diff
@@ -0,0 +1,117 @@
+    assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [([1, 2, 0], '3'), ([-1, 1, 0], '0'), ([0], '0')])
+def test_numbers_sum(input_param, expected, capsys):
+    with patch('utils.number_validation_input', side_effect=input_param):
+        numbers_sum()
+        captured = capsys.readouterr()
+        assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('numbers, operator, expected', [([2, 3], '+', '5'), ([2, 1], '-', '1'),
+                                                         ([6, 3], '/', '2.0'), ([5, 5], '*', '25'),
+                                                         ([10, 3], 'mod', '1'), ([3, 2], 'pow', '9'),
+                                                         ([10, 3], 'div', '3')])
+def test_calculator(numbers, operator, expected, capsys):
+    with patch('utils.number_validation_input', side_effect=numbers), \
+            patch('builtins.input', return_value=operator):
+        calculator()
+        captured = capsys.readouterr()
+        assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [(1000, '1000 - Volodymyr Vernadsky'),
+                                                   (3, 'nominal doesn`t exist')])
+def test_persons_on_money(input_param, expected, capsys):
+    person_on_money(input_param)
+    captured = capsys.readouterr()
+    assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [('a1', 'cell is black'),
+                                                   ('a2', 'cell is white'),
+                                                   ('string', 'incorrect cell')])
+def test_cell_color(input_param, expected, capsys):
+    with patch('builtins.input', return_value=input_param):
+        cell_color()
+        captured = capsys.readouterr()
+        assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [(10, '10 is 1010\n1010 is 10'),
+                                                   (135, '135 is 10000111\n10000111 is 135')])
+def test_decimal_to_binary(input_param, expected, capsys):
+    decimal_to_binary(input_param)
+    captured = capsys.readouterr()
+    assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, computer_choise, expected',
+                         [(['paper', 'n'], 'rock', 'you choose paper, computer choose rock...\nyou win!'),
+                          (['paper', 'n'], 'scissors', 'you choose paper, computer choose scissors...\nyou lose!'),
+                          (['paper', 'n'], 'paper', 'you choose paper, computer choose paper...\ndraw!')])
+def test_rock_paper_scissors(input_param, computer_choise, expected, capsys):
+    with patch('random.choice', return_value=computer_choise), patch('builtins.input', side_effect=input_param):
+        rock_paper_scissors()
+        captured = capsys.readouterr()
+        assert captured.out.strip() == expected
```

Comment:
tests are great.
but better to return results and print them in main.
in such case you could avoid useng capsys all the time.
it's bad practise